proto/internal/temporal/server/api/matchingservice/v1/request_response.proto

```diff
@@ -561,4 +561,19 @@ message ListWorkersRequest {
 message ListWorkersResponse {
     repeated temporal.api.worker.v1.WorkerInfo workers_info = 1;
     bytes next_page_token = 2;
+}
+
+// (-- api-linter: core::0134::request-resource-required=disabled
```

Comment:
this api linter stuff is getting absurd, I almost want to suggest naming it ConfigureTaskQueue to avoid this crap :disappointed: